### DISCUSSION TITLE
where is official helm chart docs?

### DISCUSSION THREAD

**Discussion by maplewf** [crossplane/crossplane#5271] [Q&A] [answered]
Hi,

I'm new for crossplane and start to use it via helm, but I can't find official usage document for its helm chart. Anyone know about that?

best regards

**Comment by jbw976** [comment, marked as answer]
This section of the docs should hopefully be helpful for you! 🙏 
https://docs.crossplane.io/latest/software/install/#customize-the-crossplane-helm-chart 

note the "All Crossplane customization options" box that can be expanded to show every possible option of the helm chart 😂

**Reply by maplewf** [reply]
Thanks, I actually found this page before but not noticed those helm values parameter option which is folded by default.

**Reply by jbw976** [reply]
Ah yes, I have had problems with noticing those boxes myself multiple times.  I added a link to your trouble with them as well 😉 
https://github.com/crossplane/docs/issues/480#issuecomment-1903143145

**Reply by phisco** [reply]
FYI: You can see the same exact content in the default values.yaml file itself, you can get them with `helm show values crossplane-stable/crossplane`
